### DISCUSSION TITLE
Lidemy 平台課程

### DISCUSSION THREAD

**Discussion by ming320003** [Lidemy/forum#39] [General]
最近在老師的粉絲專業上看到了關於程式導師實驗計畫的時間延後了。
想問老師假如最後去報名了，但是卻落選的話，單純照著
https://lidemy.com/p/lidemy-subscription 上的順序來訂閱跟課，也全部跟完了的話，有辦法像是計畫一樣找到軟體工程師的工作嗎？
(承上) 在同一個網頁上有說課程會有相對來說不完整的地方。想請問會因為這樣而無法順利銜接(課程和課程間)嗎？
我覺得延後日期可以讓我比較有比較多的緩衝來學到一個程度，但是也會因此有些擔心，(因為會比較慢才知道結果)，假設我繼續在平台學習，學到四月中的結果是沒有入選，大概會有點慌？
所以想問問看(才能讓我知道自己在幹嘛)。
謝謝老師

**Comment by aszx87410** [comment]
> 單純照著
https://lidemy.com/p/lidemy-subscription 上的順序來訂閱跟課，也全部跟完了的話，有辦法像是計畫一樣找到軟體工程師的工作嗎？

剛好今天看到一篇文章，推薦給你：https://jimmyswebnote.com/change-career-to-front-end-developer/

其實程式導師實驗計畫會看的課程平台上幾乎都有了，差別就只在於有社群然後有助教改作業以及有作業可以寫，作業這個對於實作能力會差滿多的，但如果自學然後自己寫作業其實也 ok

以技能的部分我覺得再補充幾個課程就 ok 了

> 在同一個網頁上有說課程會有相對來說不完整的地方。想請問會因為這樣而無法順利銜接(課程和課程間)嗎？

有可能，有些課程可能會覺得怎麼突然跳到這邊之類的，需要其他額外資源去補齊

> 我覺得延後日期可以讓我比較有比較多的緩衝來學到一個程度，但是也會因此有些擔心，(因為會比較慢才知道結果)，假設我繼續在平台學習，學到四月中的結果是沒有入選，大概會有點慌？

基本上如果能繼續保持學習一直到報名的時候（二三月），那錄取機率其實會很高，因為你有證明你本來就有自學的能力，所以這點不用太過於擔心

**Reply by ming320003** [reply]
謝謝老師的回覆！
我會好好參考那篇文章。
